pages/docs/react/latest/components-and-props.mdx

```diff
@@ -116,9 +116,20 @@ function Greeting(props) {
 
 In JSX, you can apply optional props with some special syntax:
 
+<div className="hidden">
```

Comment:
Should remove?

Reply:
Moved the prelude up a bit and mentioned that the following examples will utilize this one so I could get rid of the hidden class.